.github/workflows/ci.yml

```diff
@@ -0,0 +1,33 @@
+name: ci
+
+on: pull_request
+
+jobs:
+
+  backend:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Fetch tags and commits needed for setuptools-git-version
+        run: |
+          git fetch --depth=1 origin +refs/tags/*:refs/tags/*
+          git fetch origin ${{ github.head_ref }} && git checkout ${{ github.head_ref }}
+          git describe --tags --long --dirty
+        # If the above git describe command doesn't work,
+        # then setuptools-git-version will also fail.
+
+      - name: Set up Python
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.6
+
+      - name: Install Python dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install tox
+
+      - name: Run back-end tests
+        run: |
+          tox -e lint,py36-md32
```

Comment:
Could you set this up as a matrix that runs against Python 3.6 and Python 3.8? Travis was running `lint`, `py36-md32`, and `py38-md32`.

Reply:
Done